docs/README.md

```diff
@@ -1,28 +1,152 @@
+[stack-azure](https://github.com/crossplaneio/stack-azure). 
+
+### Infrastructure Stacks 
+Infra Stacks like
+[stack-gcp](https://github.com/crossplaneio/stack-gcp),
+[stack-aws](https://github.com/crossplaneio/stack-aws), and
+[stack-azure](https://github.com/crossplaneio/stack-azure) extend the Kubernetes
+API to support managed service provisioning (DBaaS, cache, buckets), secure
+connectivity (VPCs, subnets, peering, ACLs, secrets), and provisioning managed
+Kubernetes clusters on demand to further isolate the blast radius of
+applications. 
+
+Infra Stacks are pre-built and published to the [Stacks
+registry](https://hub.docker.com/search?q=crossplane&type=image) where they can
+be installed by a Cluster Admin using a
+[`ClusterStackInstall`](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#installation-flow)
+Kubernetes API kind or with the [`stack
+install`](https://github.com/crossplaneio/crossplane-cli#usage) command that
+installs a Stacks with permissions to watch resources at a cluster level, across
+all namespaces.
 
-The full architecture and vision of the Crossplane project is described in depth in the [architecture document](https://docs.google.com/document/d/1whncqdUeU2cATGEJhHvzXWC9xdK29Er45NJeoemxebo/edit?usp=sharing). It is the best place to learn more about how Crossplane fits into the Kubernetes ecosystem, the intended use cases, and comparisons to existing projects.
+Infra Stacks are covered in both the [Using Services](using-services.md) and
+[Using Stacks](using-stacks.md) guides. 
+
+### Application Stacks 
+App Stacks depend on Infra Stacks like
+[stack-gcp](https://github.com/crossplaneio/stack-gcp),
+[stack-aws](https://github.com/crossplaneio/stack-aws), or
+[stack-azure](https://github.com/crossplaneio/stack-azure) to provide managed
+services via the Kubernetes API.
+
+App Stacks are pre-built and published to the [Stacks
+registry](https://hub.docker.com/search?q=crossplane&type=image) where they can
+be installed by an app team using a
+[`StackInstall`](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#installation-flow)
+Kubernetes API kind or with the [`stack
+install`](https://github.com/crossplaneio/crossplane-cli#usage) command that
+installs Stacks with permissions only within a single namespace.
+
+The [Using Stacks](using-stacks.md) guide covers how to deploy a [portable
+Wordpress App Stack](https://hub.docker.com/r/crossplane/sample-stack-wordpress)
+into multiple clouds using Infra Stacks for
+[stack-gcp](https://github.com/crossplaneio/stack-gcp),
+[stack-aws](https://github.com/crossplaneio/stack-aws),
+[stack-azure](https://github.com/crossplaneio/stack-azure). 
+
+### Build your own Stack 
+The [Developer Guide](developer-guide.md) shows how to
+build and extend Infra Stacks and App Stacks. 
+
+## Crossplane Workloads 
+Crossplane includes a workload scheduler that can factor
```

Comment:
This is another area in which our claims don't match reality. Crossplane includes one scheduler implementation that:

1. Watches for unscheduled `KubernetesApplications`.
1. Gets all `KubernetesCluster` resource claims that match the `KubernetesApplication`'s cluster selector.
1. Picks a `KubernetesCluster` using a simple round-robin strategy.

https://github.com/crossplaneio/crossplane/blob/80d4fde87029857/pkg/controller/workload/kubernetes/scheduler/scheduler.go#L69